### PR TITLE
Removed hardhat specific config from deploy-local.js

### PR DESCRIPTION
**Pull request by dave**
Hey @pheuberger @kargakis this is a startup script for the docker-compose stuff I'm working on...

### CODE REVIEW

**Review by kargakis — APPROVED** [review]

File: scripts/startup-local.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+echo "Hardhat is now ready!"
+
+echo "Deploying contracts (logs in /var/log/deploy.log)"
+yarn hardhat run scripts/deploy-local.js --network localhost > /var/log/deploy.log
+if ! grep '==== config data ====' /var/log/deploy.log >/dev/null ; then
+  echo "Deploy failed (config data not found in deploy logs). Exiting."
+  exit 1
+fi
+
+echo "Creating config file ($1):"
```
Comment (pheuberger) [inline scripts/startup-local.sh:20]: do we have documentation how to call this script? seeing it's taking the config file path as an argument

File: scripts/startup-local.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+grep -A100 '==== config data ====' /var/log/deploy.log | tail -n +2 > $1
+
+echo "Creating healthcheck flag /var/log/done.flag"
+touch /var/log/done.flag
```
Comment (pheuberger) [inline scripts/startup-local.sh:24]: where is this flag being used?

File: scripts/startup-local.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+echo "Finished!"
+while :;do
```
Comment (pheuberger) [inline scripts/startup-local.sh:27]: a little more elegant solution than the loop:

```shell
yarn hardhat node --network hardhat &
PID=$!
...
rest of script
...
echo "Finished!"
wait $PID
```

File: scripts/startup-local.sh (hunk 1 above)
Comment (dave) [reply scripts/startup-local.sh:31]: See: https://github.com/rnd-inc/toucan-bridge-service/pull/80#discussion_r913913550

File: scripts/startup-local.sh (hunk 2 above)
Comment (dave) [reply scripts/startup-local.sh:35]: See: https://github.com/rnd-inc/toucan-bridge-service/pull/80#discussion_r913920662

**Review by pheuberger — COMMENTED** [review]
great work! one thing, can you squash this last commit into the previous one? IMHO it's not a really good stand-alone commit, especially not with that commit message 😁

**Comment by dave** [comment]
@pheuberger Done!

**Comment by dave** [comment]
Hey @pheuberger don't merge this yet - I think I've decided that this shell script shouldn't live here... Give me a few minutes to work out what I'm going to do.

**Comment by pheuberger** [comment]
no problem. just let me know when you're good to merge :)

**Comment by dave** [comment]
@pheuberger I moved the hardhat specific shell script into the other repo, so this can merge now.

**Review by kargakis — APPROVED** [review]

**Pull request merged**